sdk/documenttranslation/azure-ai-documenttranslation/README.md

```diff
@@ -1,35 +1,402 @@
+ - Identifying supported glossary and document formats
+
+### Input
+
+Input to create a translation job requires that a list of `DocumentTranslationInput` be passed. A single source URL to documents can
+be translated to many different languages:
+
+```python
+from azure.ai.documenttranslation import DocumentTranslationInput, TranslationTarget
+
+my_input = [
+    DocumentTranslationInput(
+        source_url="<sas_url_to_source>",
+        targets=[
+            TranslationTarget(target_url="<sas_url_to_target_fr>", language_code="fr"),
+            TranslationTarget(target_url="<sas_url_to_target_de>", language_code="de")
+        ]
+    )
+]
+```
+
+Or multiple different sources can be provided each with their own targets.
+
+```python
+from azure.ai.documenttranslation import DocumentTranslationInput, TranslationTarget
+
+my_input = [
+    DocumentTranslationInput(
+        source_url="<sas_url_to_source_A>",
+        targets=[
+            TranslationTarget(target_url="<sas_url_to_target_fr>", language_code="fr"),
+            TranslationTarget(target_url="<sas_url_to_target_de>", language_code="de")
+        ]
+    ),
+    DocumentTranslationInput(
+        source_url="<sas_url_to_source_B>",
+        targets=[
+            TranslationTarget(target_url="<sas_url_to_target_fr>", language_code="fr"),
+            TranslationTarget(target_url="<sas_url_to_target_de>", language_code="de")
+        ]
+    ),
+    DocumentTranslationInput(
+        source_url="<sas_url_to_source_C>",
+        targets=[
+            TranslationTarget(target_url="<sas_url_to_target_fr>", language_code="fr"),
+            TranslationTarget(target_url="<sas_url_to_target_de>", language_code="de")
+        ]
+    )
+]
+```
+
+> Note: the target_url for each target language must be unique.
+
+See the service documentation for all [supported languages][supported_languages].
+
+### Return value
+
+There are primarily two types of return values when checking on the result of a translation job - `JobStatusResult` and `DocumentStatusResult`.
+* A `JobStatusResult` will contain the details of the entire job, such as it's status, ID, any errors, and status summaries of the documents in the job.
+* A `DocumentStatusResult` will contain the details of an individual document, such as it's status, translation progress, any errors,
+and the URLs to the source document and translated document.
+
+## Examples
+
+The following section provides several code snippets covering some of the most common Document Translation tasks, including:
+
+* [Translate your documents](#translate-your-documents "Translate Your Documents")
+* [Check status on individual documents](#check-status-on-individual-documents "Check Status On Individual Documents")
+* [List translation jobs](#list-translation-jobs "List Translation Jobs")
+
+### Translate your documents
+Translate the documents in your source container to the target containers, using custom glossaries.
```

Comment:
thinking about progressive disclosure and how this is the first time a potential user will see code, I wonder if you should remove glossaries from here and leave this as simple as possible.
In the samples folder, there can def be one sample with the glossaries.

Reply:
agree